scripts/deployThalesRoyale/thalesRoyale/changeSafeBoxRoyaleSeason.js

```diff
@@ -33,14 +34,20 @@ async function main() {
     /* ========== PROPERTIES ========== */
 
 	const safeBoxPercentage = 5; // CHANGE for percntage
-	const safeBox = owner.address // CHANGE for address
 
     /* ========== SAFE BOX FOR ROYALE ========== */
 
 	const ThalesRoyale = await ethers.getContractFactory('ThalesRoyale');
 	const thalesRoyaleAddress = getTargetAddress('ThalesRoyale', network);
 	console.log('Found ThalesRoyale at:', thalesRoyaleAddress);
 
+	if (networkObj.chainId == 10 || networkObj.chainId == 69) {
+		SafeboxAddress = getTargetAddress('SafaBox', network);
+	} else {
+		const Safebox = snx.getTarget({ network, contract: 'SafaBox' });
+		SafeboxAddress = Safebox.address;
```

Comment:
are you sure this works? 
Last time got outdated addresses using the snx library

Reply:
what does SNX have to do with SafeBox?

Reply:
removed